src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -85,6 +85,26 @@
                             "type": "boolean",
                             "default": true
                         },
+                        "nullEqualsEmpty": {
+                            "description": "When set to false, this flag indicates that null and empty list are not equal, and changing the values from null to empty list would indicate diff",
+                            "type": "boolean",
+                            "default": false
+                        },
+                        "caseSensitive": {
+                            "description": "When set to true, this flag indicates that case of the string will be honored, and changing the case of the string would indicate diff",
+                            "type": "boolean",
+                            "default": true
+                        },
```

Comment:
Don't think we have enough data to make an informed decision to choose these default values yet. Can we postpone specifying the default value to encourage explicit schematization for now?